app/workers/evss/disability_compensation_form/job_status.rb

```diff
@@ -1,198 +1,15 @@
 # frozen_string_literal: true
 
+require 'sidekiq/form526_job_status_tracker/job_tracker'
```

Comment:
Could we remove this and the metrics class under this directory entirely if we just included those modules in EVSS::DisabilityCompensationForm::Job?